test/unit/errorHandling.test.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { expect } from 'chai';
+import { SfError } from '@salesforce/core';
+import { errorIsGack, errorIsTypeError } from '../../src/errorHandling';
+
+describe('typeErrors', () => {
+  let typeError: Error;
+
+  before(() => {
+    try {
+      const n = null;
+      // @ts-expect-error I know it's wrong, I need an error!
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+      n.f();
+    } catch (e) {
+      if (e instanceof TypeError) {
+        typeError = e;
+      }
+    }
+  });
+  it('matches on TypeError as error name', () => {
+    expect(errorIsTypeError(typeError)).to.be.true;
+  });
+
+  it('matches on TypeError in ', () => {
+    expect(errorIsTypeError(typeError)).to.be.true;
```

Comment:
This is the same test as the one above it.

Reply:
I meant to come back and finish that.  Nice catch!